nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/magick/utils/ImageResizer.java

```diff
@@ -83,14 +83,15 @@ public static ImageInfo resize(String inputFile, String outputFile, int targetWi
      * Adapts width and height to a max conserving ratio.
      *
      * @since 10.3
+     * @return new Point to scale or the original one if none is > max.
```

Comment:
Would it be interesting to clean up the NPE checks in the callers, since the contract is now to never return `null`?
- https://github.com/nuxeo/nuxeo/blob/master/nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/magick/utils/ImageResizer.java#L65
- https://github.com/nuxeo/nuxeo/blob/master/nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/test/java/org/nuxeo/ecm/platform/picture/core/im/TestImageResizer.java#L42

Reply:
You're completely right! I'm doing it right now. Thanks for the review!